Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerEngine.ps1

```diff
@@ -1495,6 +1495,64 @@
         }
     }
 
+    Function Test-DownloadDomainsConfiguration {
+        param(
+            [Parameter(Mandatory = $true)][object]$OwaVDirObject
+        )
+        Write-VerboseOutput("Calling: Test-DownloadDomainConfiguration")
+
+        <#
+        Unknown 0
+        Download Domains disabled 1
+        Download Domains enabled and configured as expected 2
+        Download Domains enabled and external download host name = internal/external owa url 4
+        Download Domains enabled but external download host name not set 8
+        Download Domains enabled and internal download host name = internal/external owa url 16
+        Download Domains enabled but internal download host name not set 32
+        #>
+
+        $downloadDomainsStatus = 0
+
+        try {
+            $downloadDomainsEnabled = (Get-OrganizationConfig -ErrorAction Stop).EnableDownloadDomains
```

Comment:
This is the main reason why this function shouldn't be in the `Invoke-AnalyzerEngine`, we shouldn't have any 'get' type commands that needs to be run on the customer's environment. 

We are already running `Get-OrganizationConfig` in `Get-ExchangeInformation`. Add two attributes to the `HealthChecker.ExchangeInformation` class. 

- GetOrganizationConfig //store the results from Get-OrganizationConfig
- EnableDownloadDomains //quick access to the the value that you are looking for

Update the code to look something like this

```
        try {
$organizationConfig =  (Get-OrganizationConfig -ErrorAction Stop)
            $exchangeInformation.GetOrganizationConfig = $organizationConfig 
        } catch {
            Write-Yellow "Failed to run Get-OrganizationConfig."
            Invoke-CatchActions
        }

if ($null -ne $organizationConfig) {
//save values
} else {
Write-ScriptDebug "MAPI HTTP Enabled and download domains enabled results not accurate"
}
```

Reply:
Done